### DISCUSSION TITLE
Hey

### DISCUSSION THREAD

**Discussion by IceHacks** [IceHacks/SurvivCheatInjector#745] [Announcements]
Suggestions, etc. should go in new Discussions. Leave the issues in issues for now.

**Comment by ghost** [comment]
<h3>Suggestions</h3>
It would be nice if you added legitmode and smartswitch back into the cheat.

- Speaking of smartswitch, could you try merging sniper and smart switch together? 
Also please move this discussion into the Ideas category.
<hr />
Thanks!
<br />

**Reply by TylerWhat** [reply]
what is a legal mode ?

**Reply by IceHacks** [reply]
Hey! SmartSwitch has been renamed to "UseBestGun." And I am currently working on making those two work together nicely. They like to fight a lot.

**Reply by IceHacks** [reply]
I rephrased my original post. Are users able to create discussions?

**Reply by ghost** [reply]
I don't know. I haven't tried it.

**Comment by AsiaGuy** [comment]
is the hack using VPN? That's make my pc lag
